modules/gitops-inbuilt-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-inbuilt-permissions-for-cluster-config_{context}"]
+= In-built permissions for cluster configuration
+
+Argo CD has the following permissions to manage specific cluster-scoped resources which include platform operators, optional OLM operators and user management. 
+
+[NOTE]
+====
+Argo CD will not have cluster-admin permissions.
+====
+
+By default the Argo CD instance is provided with the following permissions:
```

Comment:
This feels redundant to the sentence above.

Reply:
I think this is not resolved. The new sentence reads:

> The Argo CD instance is provided with the following permissions

And the sentence just above reads:

> The default Argo CD has the following permissions